chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx

```diff
@@ -83,9 +87,9 @@ export const ChatButtonStateful = (props: IChatButtonStatefulParams) => {
 
     return (
         <ChatButton
-            componentOverrides={buttonProps?.componentOverrides}
+            componentOverrides={chatButtonProps?.componentOverrides}
             controlProps={outOfOperatingHours ? outOfOfficeControlProps : controlProps}
-            styleProps={outOfOperatingHours ? outOfOfficeStyleProps : buttonProps?.styleProps}
+            styleProps={outOfOperatingHours ? outOfOfficeStyleProps : chatButtonProps?.styleProps}
```

Comment:
what will happen if chatButtonProps is empty?, dont we need a default props?

Reply:
default props are defined in the component layer. Passing null here is acceptable